src/XmlApiTestCase.php

```diff
@@ -60,6 +60,23 @@ private function assertXmlHeader(Response $response)
      */
     private function assertXmlResponseContent(Response $response, $filename)
     {
-        parent::assertResponseContent($response, $filename, 'xml');
+        $actualResponse = $response->getContent();
```

Comment:
Anyway, if it would be `$actualResponseContent` then the method parameter `$response` should be renamed to `$actualResponse`.